ray/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Ray
+
+## 1.0.0 / 2023-08-01
```

Comment:
Probably should be `## Unreleased` here since the version should be 0.0.1 right now.

Reply:
This is the new default template. The release command will automatically update the date and leave the rest of this as it is when the RM will use the `--new` option to release the integration